sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -516,6 +505,35 @@ case class DescribeTableCommand(
       new MetadataBuilder().putString("comment", "comment of the column").build())()
   )
 
+  def describeSchema(
```

Comment:
How about this?
```

  protected def describeSchema(schema: StructType, header: Boolean): Seq[Row] = {
    val buffer = new ArrayBuffer[Row]
    if (header) {
      buffer += Row(s"# ${output.head.name}", output(1).name, output(2).name)
    }
    schema.foreach { column =>
      buffer += Row(column.name, column.dataType.simpleString, column.getComment().orNull)
    }
    buffer
  }
```

Reply:
@maropu describeSchema has two callers. One from the describeTable code path and one from describeQuery. The other caller passes the row buffer.

Reply:
Ah, I see. Can you add protected, anyway?